package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+This can potentially lead to some ambiguity in the assignment of
+donors/acceptors for residues such as histidine or cytosine.
+
+For more information about the naming convention in GLYCAM06 have a look at the
+`Carbohydrate Naming Convention in Glycam`_.
+
+.. _`Carbohydrate Naming Convention in Glycam`:
+   http://glycam.ccrc.uga.edu/documents/FutureNomenclature.htm
+
+The lists of donor and acceptor names can be extended by providing lists of
+atom names in the `donors` and `acceptors` keywords to
+:class:`WaterBridgeAnalysis`. If the lists are entirely inappropriate
+(e.g. when analysing simulations done with a force field that uses very
+different atom names) then one should either use the value "other" for `forcefield`
+to set no default values, or derive a new class and set the default list oneself::
+
+ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
+       DEFAULT_DONORS = {"OtherFF": tuple(set([...]))}
+       DEFAULT_ACCEPTORS = {"OtherFF": tuple(set([...]))}
+
+Then simply use the new class instead of the parent class and call it with
+`forcefield` = "OtherFF". Please also consider to contribute the list of heavy
+atom names to MDAnalysis.
+
+.. rubric:: References
+
+.. [Gregoret1991] L.M. Gregoret, S.D. Rader, R.J. Fletterick, and
+   F.E. Cohen. Hydrogen bonds involving sulfur atoms in proteins. Proteins,
+   9(2):99–107, 1991. `10.1002/prot.340090204`_.
+
+.. _`10.1002/prot.340090204`: http://dx.doi.org/10.1002/prot.340090204
+
+
+How to perform WaterBridgeAnalysis
+-----------------------------------
+
+All water bridges between arginine and aspartic acid can be analysed with ::
+
+  import MDAnalysis
+  import MDAnalysis.analysis.hbonds
+
+  u = MDAnalysis.Universe('topology', 'trajectory')
+  w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP', distance=3.0, angle=120.0)
+  w.run()
+
+.. Note::
+
+   Due to the way :class:`WaterBridgeAnalysis` is implemented, it is
+   more efficient to have the second selection (`selection2`) be the
+   *larger* group, e.g. the whole protein when looking at ligand-protein
+   water bridge interactions.
+
+
+The results are stored as the attribute
+:attr:`WaterBridgeAnalysis.timeseries`; see :ref:`Analysis Output` for the
+format and further options.
+
+For further data analysis it is convenient to process the
+:attr:`~WaterBridgeAnalysis.timeseries` data into a normalized table with the
+:meth:`~WaterBridgeAnalysis.generate_table` method, which creates a new data
+structure :attr:`WaterBridgeAnalysis.table` that contains one row for each
+observation of a hydrogen bond::
+
+  w.generate_table()
+
+This table can then be easily turned into, e.g., a `pandas.DataFrame`_, and
+further analyzed::
+
+  import pandas as pd
+  df = pd.DataFrame.from_records(w.table)
+
+For example, plotting a histogram of the hydrogen bond angles and lengths is as
+simple as ::
+
+  df.hist(column=["angle", "distance"])
```

Comment:
Did you try this? Does it work?

Reply:
@orbeckst generate_table() and pd.DataFrame.from_records(w.table) are working. df.hist(column=["angle", "distance"]) works but is probabily not very useful.